keyboards/pimoroni/keybow2040/config.h

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+#define TRANSIENT_EEPROM_SIZE 1024
+
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+
+// for I2C
+#define DRIVER_COUNT 1
+
+// adafruit_is31fl3731 says 0x74 is the default
+#define DRIVER_ADDR_1 0b1110100
+
+#define I2C0_CLOCK_SPEED 50000
```

Comment:
At the moment the RP2040 I2C driver is broken and does not produce proper timings, so you may need this value (it actually gives about 186 kHz in my testing).  Eventually it should be removed (the IS31FL3731 chip is supposed to support 400 kHz, but the current driver makes it about 706 kHz instead, so the chip might not work properly with the default speed configuration).

Reply:
Right. I saw a comment on another RP2040 keyboard QMK config saying that it needed a clock speed setting to work reliably. I didn’t know it was a bug in the driver, I was trying all sorts of things to see if they improved matters.